core/locale/domain/label.go

```diff
@@ -78,27 +70,27 @@ func (l *Label) SetCount(count int) *Label {
 	return l
 }
 
-// SetLocale on a label
-func (l *Label) SetLocale(localeCode string) *Label {
+// SetLocaleCode on a label
```

Comment:
This is backwards incompatible (it changes the method names).
I would be fine with doing this, but it will require extra work for users of the locale package.

Reply:
Maybe we use this already in templates - so I would not break backward here

Reply:
Added both, SetLocale is deprecated :)